src/Client.php

```diff
@@ -663,7 +715,9 @@ public function updateEmailContentInEditableSection($emailId, $htmlId, $args = a
         $args['id'] = $emailId;
         $args['htmlId'] = $htmlId;
 
-        return $this->getResult('updateEmailContentInEditableSection', $args, false, $returnRaw);
+        return new UpdateEmailContentInEditableSectionResponse(
+            json_decode($this->post(sprintf('rest/asset/v1/email/%d/content/{htmlId}.json', $emailId), $args)->getBody())
```

Comment:
old code used to return `->getBody(true)` is that bool not required anymore?

Reply:
`->getBody()` no longer accepts parameters. This change is okay